### DISCUSSION TITLE
Title on background event

### DISCUSSION THREAD

**Discussion by shijenwai** [vkurko/calendar#304] [Q&A] [answered]
When event set to display: "background", title disappear.
Is there a way to show title on background event?

I found someone show title on background event with fullcalendar.
https://codepen.io/arshaw/pen/zYrEWaY?editors=0010

Not sure if the feaature is possible here.

**Comment by vkurko** [comment, marked as answer]
Currently, background events do not initially render any content. To change this, you need to override the rendering of all events using [eventContent](https://github.com/vkurko/calendar#eventcontent). By checking `event.display`, you can distinguish which type of event is currently being processed.

**Reply by shijenwai** [reply]
Thanks a lot! 😊
